logstash-logging-spring-boot-starter/src/main/java/pl/piomin/logging/config/SpringLoggingAutoConfiguration.java

```diff
@@ -37,6 +37,8 @@ public class SpringLoggingAutoConfiguration {
     private String requestIdHeaderName = "X-Request-ID";
     private String correlationIdHeaderName = "X-Correlation-ID";
 
+    private boolean ignorePayload = false;
```

Comment:
you need to add also getter/setter to inject value from configuration properties